keda/templates/NOTES.txt

```diff
@@ -1,6 +1,23 @@
-=====================================================================================
-KEDA - Kubernetes Event Driven Autoscaling
-=====================================================================================
+:::^.     .::::^:     :::::::::::::::    .:::::::::.                   .^.                  
```

Comment:
OH MY 😄 

I with @JorTurFer on this one, but not completely against 😄

Reply:
1 + 0,5 + 0,5 = 2 of 3, so we can merge it xD

Reply:
> Looks like we can merge it xD

You confused me =)

Is this a blocker or not? `--set asciiArt=false` disables this though

Reply:
does helm support that? or how does it work?
We can merge it

Reply:
Yes! If I would at least push the commit where I added this :)
https://github.com/kedacore/charts/pull/484/commits/0814f1b5abc511271bb92e0ce62b2fa4f60cd17a